.gitmodules

```diff
@@ -35,3 +35,9 @@
 	path = tpls/qpp
 	url = https://github.com/softwareQinc/qpp.git
 	shallow = true
+[submodule "tpls/asio"]
```

Comment:
When I brought this up at a [prior code review](https://github.com/NVIDIA/cuda-quantum/pull/1012#discussion_r1449173665), I think Thien preferred to keep it where it was. Any objections now @1tnguyen ?

Reply:
We should use submodules for these things.